recipe/meta.yaml

```
@@ -334,8 +372,11 @@ about:
   description: |
     CUDA Runtime Native Libraries
   doc_url: https://docs.nvidia.com/cuda/index.html
+  dev_url: https://docs.nvidia.com/cuda/index.html
 
 extra:
+  skip-lints:
+    - license_file_overspecified
```

Tests are missing as expected, let's tell the linter for future reference 
```suggestion
  skip-lints:
    - license_file_overspecified
    - missing_tests
```